doc/doxygen/src/Examples.dox

```diff
@@ -144,7 +144,7 @@
  * `kInfoKey_EntityReferencesMatchPrefix` key.
  *
  * @code{.py}
- * from openassetio import Context
+ * from openassetio.access import ResolveAccess
```

Comment:
[Question] Don't we still need to context import for this example?

Reply:
The `Context` class is not referenced directly (only indirectly by `createContext`) so doesn't need to be explicitly imported.